README.md

```diff
@@ -16,21 +16,22 @@ We welcome contributions! Please open a PR with additions or corrections!
 The files should start and end with a row containing exactly `---` and should contain Yaml records with the fields described below.
 For an example of a file with a formalization entry, see [Q208416.md](_thm/Q208416.md).
 
-* `wikidata`: Wikidata identifier for this theorem (or concept related to the theorem). This is usually the Wikipedia page containing the theorem.
+* `wikidata`: Wikidata identifier for this theorem (or concept related to the theorem). Valid identifiers start with the latter Q followed by a number.
 * `id_suffix` (optional): disambiguates an entry when two theorems have the same wikidata identifier. `X` means an extra theorem on a Wikipedia page (e.g. a generalization or special case), `A`/`B`/... means different theorems on one Wikipedia page that doesn't have a "main" theorem.
-* `msc_classification`: Our best guess of the [MSC-classification](https://msc2020.org/). Please PR a better suggestion!
-* `wikipedia_links`: The exact link used on
-* Then zero or more entries for the formalizations `isabelle`, `hol_light`, `coq`, `lean`, `metamath`, `mizar`
+* `msc_classification`: Our best guess of the [MSC-classification](https://msc2020.org/) of this theorem. Please PR a better suggestion!
+* `wikipedia_links`: list of exact wikipedia links to the relevant page(s). Each link has the format `[[Page name]]` or `[[Wikilink|Displayed name]]`.
+* Then zero or more entries for the formalizations in any of the supported proof assistants (`isabelle`, `hol_light`, `coq`, `lean`, `metamath`, `mizar`). Several formalization entries for one assistant are allowed.
 
-For each proof assistant we record the following information
+For each formalization in each proof assistant, we record the following information
 
-* `status`: `formalized`, `statement`
+* `status`: `formalized` (the proof is formalized), `statement` (just the statement is)
 * `library`: in what library does the formalization appear
-  - S standard library
-  - L main (biggest) mathematical library (afp, hol light outside standard library, mathcomp, mathlib, mml, set.mm)
-  - X external to main or standard library (e.g. a dedicated repository)
-* `url`: a URL pointing to the formalization (or a list collecting a list of theorems for a particular proof assistant).
-* `authors`:  (optional)
+  - "S": standard library
+  - "L": main (biggest) mathematical library (afp, hol light outside standard library, mathcomp, mathlib, mml, set.mm)
+  - "X": external to the main or standard library (e.g. a dedicated repository)
+* `identifiers`: list of theorems for a particular proof assistant; strongly recommended for "S" or "L" formalizations
```

Comment:
Do we want to track the identifier in this repository? I would prefer to have less information here, so that it doesn't have to be updated frequently.

Otherwise LGTM

Reply:
Updated now. Merging #6 makes more sense logically.